application/src/main/java/org/togetherjava/tjbot/commands/help/ChangeHelpTitleCommand.java

```diff
@@ -58,7 +58,7 @@ public ChangeHelpTitleCommand(@NotNull HelpSystemHelper helper) {
     public void onSlashCommand(@NotNull SlashCommandInteractionEvent event) {
         String title = event.getOption(TITLE_OPTION).getAsString();
 
-        if (!helper.handleIsHelpThread(event) || !handleIsValidTitle(title, event)) {
```

Comment:
Now it could also be used in the overview channel itself, not an option. I.e. we have to at least check that its a thread and not a regular channel.